custom_components/tuya_local/devices/pinguino_portable_airconditioner.yaml

```diff
@@ -0,0 +1,125 @@
+name: Pinguino
+products:
+  - id: anR7aeh1YPTKAz5a
+    name: Pinguino PACEL140LRFK 3ALBK
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - value: "off"
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: preset_mode
+          conditions:
+            - dps_val: null
+              value: "off"
+            - dps_val: "1"
+              value: dry
+            - dps_val: "2"
+              value: cool
+            - dps_val: "3"
+              value: fan_only
+            - dps_val: "4"
+              value: auto
+    - id: 5
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: "4"
+          value: eco
```

Comment:
I think leave this out.  "auto" HVAC mode covers what an "Eco" mode should be (more complex logic than just switching between heating and cooling based on the temperature, such as taking presence into consideration or a schedule). In that case rename preset_mode and all references to it, so it does not create a separate preset_mode selector.
Leaving it like this will result in invalid values being present, which will interfere with scene support, and possibly fill the logs up with error messages in normal use.

Reply:
Removed.